misc/testerina/modules/testerina-core/src/main/ballerina/annotations.bal

```diff
@@ -32,6 +32,7 @@ public type TestConfig record {
 };
 
```

Comment:
Shall we check all the attributes and replace `Name of the function` with `Function` as we are using function pointers now?

Reply:
Changed where applicable